client/src/components/Dashboard/Dashboard.global.css

```diff
@@ -12,3 +12,7 @@ body.has-loaded-dashboard {
   /* HACK: Should instead (ask Design) have a responsive sidebar */
   --min-width: 730px; /* just before My Tickets Number column value overflows */
 }
+
+.has-common-sidebar {
+  padding-left: 0px;
+}
```

Comment:
**Concern.**
Is there a disadvantage to instead having this style change be in the common Sidebar component CSS module? Is the styles only for nested Sidebars? (In which case, a React parameter could describe it as nested or not, then JSX could apply a conditional class that adds this style.)

**Proposal.**

<details><summary><strong>If</strong> this <code>padding-left: 0</code> is <em>necessary</em> and <em>suitable</em> as a <strong>global</strong> style (see doubt, above)...</summary>

...<strong>then</strong> set it less often in the markup:

1. Use `bodyClassName="has-common-sidebar"`,
    instead of `contentClassName="has-common-sidebar"`.
2. Update this selector to still select Section content,
    perhaps `body.has-common-sidebar .workbench-wrapper .nav`.
 
> Disclaimer: I haven't fully thought this through, so you may need to dig and test a bit.

</details>

Reply:
The padding issue comes up when we have the `Sidebar` nested inside a `Section` since a `Section` by default has some left padding. I can look into what you have suggested.

Reply:
Hrm, given that data, I offer this additional snippet:

**`Sidebar.module.css`**
```
:global(body.has-common-sidebar) the-thing-that-needs-padding-tweak {
    padding-left: 0;
}
```

Moving class to body reduces number of times class is added to `.jsx`. Yet, the styling can still be in Sidebar's stylesheet. _Note: That `:global(...)` trick is not native CSS, it's CSS module syntax._